src/web/mod.rs

```diff
@@ -50,21 +58,23 @@ pub fn run(soup: Blender) -> HttpResult<Listening> {
     };
 
     for (path, ep) in ins.into_iter() {
-        use std::sync::Mutex;
-        let put = Mutex::new(ep.f);
+        // use std::sync::Mutex;
+
         let args = ep.arguments;
+
+        let put = Mutex::new(Box::new(ep.mutator));
```

Comment:
Can you leave this line above `let args = ep.arguments` like it was to simplify the diff?